### PR TITLE
Allow `mailaliases` to be managed via hiera

### PR DESCRIPTION
**Pull request by peelman**
#### Pull Request (PR) description

Allow mailaliases to be managed via hiera

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Review by kenyon — APPROVED** [review]

**Comment by kenyon** [comment]
I should have squashed the commits, didn't notice until I hit the merge button :(

**Comment by peelman** [comment]
> I should have squashed the commits, didn't notice until I hit the merge button :(

sorry for the mess

**Pull request merged**
